### ISSUE TITLE
Implement a backup Cache API

### ISSUE THREAD

**Issue by mjovanc** [kryptokrona/xkr-website#36]
We need a backup Cache API so we don't face down time on the website such as Novastack Cache API. Currently it gives problem on website.

**Comment by TechyGuy17** [comment]
I would say 3 different caches. 

1) Official
2) Novastack
3) One from a different provider incase the issue is the novastack infrastructure (Something like norpool)

**Comment by Coffeboi** [comment]
![17F789C1-42AE-4CF1-8E5B-B6A341E9166C](https://user-images.githubusercontent.com/84473858/186480934-ed2df424-e2d0-40ad-b4bf-b7614265fb77.png) 
Need backup

**Comment by Coffeboi** [comment]
> 

Yeah Norpool can be a nice #3. It’ll probably be up and running sometime today.
